lambdas/service/app.py

```diff
@@ -319,19 +319,28 @@ def validate_params(query_params: Mapping[str, str],
 @app.route('/integrations', methods=['GET'], cors=True)
 def get_integrations():
     query_params = app.current_request.query_params or {}
-    validate_params(query_params, entity_type=str, integration_type=str)
+    validate_params(query_params, entity_type=str, integration_type=str, entity_ids=str)
+    entity_ids = query_params.get('entity_ids', '')
+    entity_ids = None if entity_ids == '' else set(entity_ids.split(','))
     try:
         entity_type = query_params['entity_type']
         integration_type = query_params['integration_type']
     except KeyError:
         raise BadRequestError('Parameters entity_type and integration_type must be given')
-    body = _fetch_integrations(entity_type, integration_type)
+    body = _fetch_integrations(entity_type, integration_type, entity_ids)
     return Response(status_code=200,
                     headers={"content-type": "application/json"},
                     body=json.dumps(body))
 
 
-def _fetch_integrations(entity_type, integration_type):
+def _fetch_integrations(entity_type: str, integration_type: str, entity_ids: Set) -> List:
```

Comment:
Set of what? List of what? This is not the first time I point this out. There is a reason why the types in `typing` are generic.